sample-files/gen_demo.py

```
@@ -381,6 +381,84 @@ def no_grease(lst):
+    / IP(src=CLIENT_IP, dst=PAYLOAD_TARGET)
+    / TCP(sport=59012, dport=80, flags="A",
+          seq=1001 + len(jwt_req), ack=5001 + len(jwt_resp)),
+]
+packets += pkts_jwt
+
+# --- sql_injection_probe: SQL injection pattern in HTTP GET (TCP dstPort 80) ---
+# Use raw bytes (not URL-encoded) so the regex can match the literal characters.
+packets += tcp_flow(CLIENT_IP, PAYLOAD_TARGET, 59013, 80,
+    b"GET /search?q=' OR '1'='1&type=user HTTP/1.1\r\nHost: target.example.com\r\n\r\n")
+
+print(f"Payload-inspection demo: 9 flows (5 payload_contains + 4 payload_regex) added")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)